op-e2e/op_geth.go

```diff
@@ -90,7 +116,10 @@ func NewOpGeth(t *testing.T, ctx context.Context, cfg *SystemConfig) (*OpGeth, e
 	)
 	require.Nil(t, err)
 
-	l2Client, err := ethclient.Dial(node.HTTPEndpoint())
+	l2Client, err := ethclient.Dial(HTTPEndpoint)
+	require.Nil(t, err)
```

Comment:
Nit: The existing test is using `require.Nil(t, err)`, so maybe it's best to stick with it, but the more idiomatic assertion would be `require.NoError(t, err)`.